src/parcsr_ls/par_interp_trunc_device.c

```diff
@@ -229,7 +229,11 @@ hypreGPUKernel_InterpTruncationPass1_v1( hypre_DeviceItem &item,
    /* 2. save the largest max_elmts entries in sh_val/pos */
    const HYPRE_Int nt = hypre_gpu_get_num_threads<1>(item);
    const HYPRE_Int tid = hypre_gpu_get_thread_id<1>(item);
+#if defined(HYPRE_USING_SYCL)
+   HYPRE_Int *shared_mem = (HYPRE_Int*) shmem_ptr;
```

Comment:
With SYCL, we can only use "dynamic" shared memory, right? I remembered seeing it somewhere. Just to learn this. Thanks!

Reply:
Yes, as far as I know, there is only dynamic shared memory. The only way I know to declare shared memory is through the use of a sycl accessor (this is a special sycl concept that I otherwise don't much use) with the sycl::target::local option.